### PR TITLE
Scala Style Changes as needed

### PR DESCRIPTION
**Pull request by srachakonda**

### CODE REVIEW

File: src/main/scala/com/imaginea/Main.scala [hunk 1]
```diff
@@ -822,10 +830,12 @@ object Main extends App {
             mayBeFile match {
               case Some(fileName) => accum + ((name, value))
               case None =>
-                if (name.equalsIgnoreCase("userName") || name.equalsIgnoreCase("passPhase"))
+                if (name.equalsIgnoreCase("userName") || name.equalsIgnoreCase("passPhase")) {
                   accum + ((name, value))
-                else
+                }
+                else {
```
Comment (eklavya) [inline src/main/scala/com/imaginea/Main.scala]: should be "} else {"

File: src/main/scala/com/imaginea/Main.scala [hunk 2]
```diff
@@ -1496,12 +1508,16 @@ object Main extends App {
...
+  def getOrCreateKeyPair(keyName: String, keyMaterial: String, keyFilePath: Option[String], status: KeyPairStatus, defaultUser: Option[String],
+                         passPhase: Option[String]): KeyPairInfo = {
+    val mayBeKeyPair = Neo4jRepository.getSingleNodeByLabelAndProperty("KeyPairInfo", "keyName", keyName).
+      flatMap(node => KeyPairInfo.fromNeo4jGraph(node.getId))
 
     mayBeKeyPair match {
       case Some(keyPairInfo) =>
-        KeyPairInfo(keyPairInfo.id, keyName, keyPairInfo.keyFingerprint, keyMaterial, if (keyFilePath.isEmpty) keyPairInfo.filePath else keyFilePath, status, if (defaultUser.isEmpty) keyPairInfo.defaultUser else defaultUser, if (passPhase.isEmpty) keyPairInfo.passPhrase else passPhase)
+        KeyPairInfo(keyPairInfo.id, keyName, keyPairInfo.keyFingerprint, keyMaterial,
+          if (keyFilePath.isEmpty) keyPairInfo.filePath else keyFilePath, status,
```
Comment (eklavya) [inline src/main/scala/com/imaginea/Main.scala]: introduce intermediate vals

File: src/main/scala/com/imaginea/Main.scala [hunk 3]
```diff
@@ -1515,13 +1531,14 @@ object Main extends App {
...
-    val node = keyPairInfo.toNeo4jGraph(KeyPairInfo(keyPairInfo.id, keyPairInfo.keyName, keyPairInfo.keyFingerprint, keyPairInfo.keyMaterial, Some(filePath), keyPairInfo.status, keyPairInfo.defaultUser, keyPairInfo.passPhrase))
+    val node = keyPairInfo.toNeo4jGraph(KeyPairInfo(keyPairInfo.id, keyPairInfo.keyName, keyPairInfo.keyFingerprint, keyPairInfo.keyMaterial,
+      Some(filePath), keyPairInfo.status, keyPairInfo.defaultUser, keyPairInfo.passPhrase))
     KeyPairInfo.fromNeo4jGraph(node.getId)
   }
 
   def getKeyFilesDir: String = s"${Constants.getTempDirectoryLocation}${Constants.FILE_SEPARATOR}"
 
-  def getKeyFilePath(keyName: String) = s"$getKeyFilesDir$keyName.pem"
+  def getKeyFilePath(keyName: String): String = s"$getKeyFilesDir$keyName.pem"
```
Comment (eklavya) [inline src/main/scala/com/imaginea/Main.scala]: why are these functions and not vals?

File: src/main/scala/com/imaginea/activegrid/core/models/Instance.scala [hunk 4]
```diff
@@ -97,77 +101,77 @@ object Instance {
         val launchTime: Option[Long] = Some(LaunchTime)
         //map.get("launchTime").get.toString.toLong
 
-        val relationship_info1 = "HAS_storageInfo1"
+        val storeageInfoRelation1 = "HAS_storageInfo1"
```
Comment (eklavya) [inline src/main/scala/com/imaginea/activegrid/core/models/Instance.scala:110]: why do we still have relation in all the names, is it increasing readability?

Comment (srachakonda) [reply src/main/scala/com/imaginea/activegrid/core/models/Instance.scala:110]: Yeah Saurabh just to have code readability

File: src/main/scala/com/imaginea/Main.scala (hunk 1 above)
Comment (srachakonda) [reply src/main/scala/com/imaginea/Main.scala]: Format Fixed

**Pull request merged**